lib/core/src/display/render/css3d/html/html_renderer.rs

```diff
@@ -30,12 +31,13 @@ mod js {
     extern "C" {
         pub fn set_object_transform(dom:&JsValue, matrix_array:&Object);
         pub fn setup_perspective(dom: &JsValue, znear: &JsValue);
-        pub fn setup_camera_transform
+        pub fn setup_camera_orthographic(dom:&JsValue, matrix_array:&JsValue);
+        pub fn setup_camera_perspective
         ( dom          : &JsValue
-        , znear        : &JsValue
-        , half_width   : &JsValue
-        , half_height  : &JsValue
-        , matrix_array : &JsValue
+          , y_scale      : &JsValue
+          , half_width   : &JsValue
+          , half_height  : &JsValue
+          , matrix_array : &JsValue
```

Comment:
I believe the previous spacing was correct! :)

Reply:
Auto indentation :/